tests/CMakeLists.txt

```diff
@@ -12,23 +12,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set(CMAKE_FOLDER "${CMAKE_FOLDER}/tests")
 
 find_package(GTest REQUIRED CONFIG)
-include(GoogleTest)
 
-add_executable(tests_util)
+add_executable(vul_tests)
```

Comment:
An I am not planning to have a single test executable.

Reply:
?

Reply:
Not sure what you are trying to say here.